test/SPIRV/useless_word.spt

```diff
@@ -0,0 +1,35 @@
+119734787 65536 393230 11 0 
+2 Capability Addresses 
+0 Capability 
```

Comment:
Just out of curiocity. Which instruction causes the issue?
```
0 Capability
```
or 
```
0 Nop
```
Both?

Reply:
Both. By the way, shall we name it "redundant_word.spt"?

Reply:
Sound good to me.
Feel free to update the request.
I'll let Alexey S and Sam to take a look before the merge.
Thanks!